raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -224,60 +226,70 @@ def get_pending_transfer_pairs(
     return pending_pairs
 
 
-def _fee_for_payer_channel(
-    channel: NettingChannelState, amount: PaymentWithFeeAmount
-) -> Optional[FeeAmount]:
-    """ Fee deducted by the mediator for an incoming channel.
+def find_intersection(fee_func: Interpolate, line: Callable[[int], float]) -> Optional[float]:
+    """ Returns the x value where both functions intersect
 
-    The `amount` is the total incoming amount without any fees deducted.
+    `fee_func` is a piecewise linear function while `line` is a straight line
+    and takes the one of fee_func's indexes as argument.
```

Comment:
Please also add an entry to the docstring saying this function can also return `None` if we don't have enough capacity